### DISCUSSION TITLE
How to disable AVX2 to be able to run on old CPU -i5 2400- that does not support it?

### DISCUSSION THREAD

**Discussion by AB0x** [ggml-org/whisper.cpp#1793] [General]
make.exe won't run, it just stops silently. After some digging it might work if I disable AVX2 for older CPUs.

**Comment by bobqianic** [comment]
> make.exe won't run

main.exe?

You can compile from source using CMake with the `-DWHISPER_NO_AVX2` flag.

**Reply by AB0x** [reply]
```
-- Building for: NMake Makefiles
CMake Error at CMakeLists.txt:3 (project):
  Running

   'nmake' '-?'

  failed with:

   no such file or directory


CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
```

Thanks for your answer, but I have two errors one related to 'nmake' and one to 'c,c++ location'
how do you suggest I proceed?

**Reply by ulatekh** [reply]
What OS are you building under? NMake is for MS Windows. Also, it sounds like you don't have NMake installed, or are running CMake from a command prompt that doesn't have the path to NMake in it.
